sbin/pfdhcplistener

```diff
@@ -46,7 +46,7 @@ use pf::config::util;
 use pf::services::util;
```

Comment:
Can you checkout this file to devel ?

This will clash with #999 which rewrites the pfdhcplistener and it will end up using the queue anyway